torch/distributed/tensor/parallel/style.py

```diff
@@ -698,3 +699,113 @@ def __repr__(self) -> str:
         tmpstr += f"use_local_output={self.use_local_output}"
         tmpstr += ")"
         return tmpstr
+
+
+class PrepareModuleInputOutput(ParallelStyle):
+    """
+    Configure the nn.Module's inputs (and outputs) to convert the input tensors (and output tensors, respectively) of the nn.Module
```

Comment:
Can we mention this is a combination of `PrepareModuleInput` and `PrepareModuleOutput`? So that it gives user a clear picture for those who used them before.